src/Components/SellingDashboardComponents/SellingNav.js

```diff
@@ -12,7 +13,7 @@ function SellingNav() {
                     <NavLink exact to='/selling/sold'>Sold</NavLink>
                     <NavLink to='/selling/messages'>Messages</NavLink>
                     <NavLink to='/selling/help'>Help</NavLink>
-                    <button>Upload Art</button>
+                    <UploadModal/>
```

Comment:
This button and the styling of it seem to have pushed all of the rest of the selling nav out of alignment...can we make the styling the same as before?

Reply:
Thank you, will make the changes.